composer.json

```diff
@@ -30,12 +30,12 @@
         "ramsey/uuid": "^4.7.3"
     },
     "require-dev": {
-        "eventsauce/test-utilities": "dev-main",
+        "eventsauce/test-utilities": "^3.3",
         "fakerphp/faker": "^1.21.0",
         "intonate/tinker-zero": "^1.2",
         "laravel/pint": "^1.6",
         "mockery/mockery": "^1.5.1",
-        "pestphp/pest": "^2.0",
+        "pestphp/pest": "^2.5",
```

Comment:
ℹ️ I'm using the latest PHPUnit XML configuration format, which includes `<source>` that came with v10.1, which is introduced by Pest 2.5